### ISSUE TITLE
Add drag drop example for vuedraggable / SortableJS

### ISSUE THREAD

**Issue by ryanhaney** [cypress-io/cypress-documentation#917]
I've found this works pretty well when using vuedraggable, which is based on SortableJS.

https://gist.github.com/ryanhaney/638af5cc81e81715cda9c41a10c56241

It took a _long_ time to get to this point. I wouldn't have gotten this far if it weren't for this gist: https://gist.github.com/superMDguy/f10d29b17ef0a473fb2700f3c72661cd

Would appreciate it if it's added to the docs.

Thanks!

**Comment by jennifer-shehane** [comment]
Yes, this may be good added to the `.trigger(https://docs.cypress.io/api/commands/trigger.html)` doc or maybe even added as a complete example in our [`cypress-example-recipes`](https://github.com/cypress-io/cypress-example-recipes) project, since it is quite a lot of code.

**Comment by ryanhaney** [comment]
I'd be happy to fork and create a PR for either or both of those if you'd like.

**Comment by ryanhaney** [comment]
Created https://github.com/cypress-io/cypress-example-recipes/pull/204. Please let me know if you need any changes. Thanks!
